recipes/musl/musl_1.1.15.oe

```diff
@@ -0,0 +1,61 @@
+inherit autotools
```

Comment:
Please add the standard boiler plate information, such as DESCRIPTION, LICENSE, HOMEPAGE.